src/components/Sidebar/LatestRecommendations.js

```diff
@@ -0,0 +1,44 @@
+import React, { PropTypes } from 'react';
+import { FormattedRelative } from 'react-intl';
+import { Link } from 'react-router-dom';
+import BodyShort from '../Story/BodyShort';
+import './LatestRecommendations.less';
+
+const renderRecommendedPost = ({ title, author, created }) => (
+  <div className="Recommendation">
+    <div className="Recommendation__text">
+      <div className="Recommendation__title">
+        <BodyShort body={title} length={80} />
```

Comment:
Title must link to the related post with this format `/:parent_permlink/@:author/:permlink` . (but stay with grey color)